modules/acquia_cms_headless/modules/acquia_cms_headless_ui/acquia_cms_headless_ui.module

```diff
@@ -0,0 +1,382 @@
+      $local_tasks[$id]['title'] = t('View');
+      $local_tasks[$id]['class'] = ViewJsonTask::class;
+      $local_tasks[$id]['entity_type_id'] = $matches[2];
+    }
+  }
+
+  $local_tasks['oauth2_token.settings_tab']['title'] = t('API Settings');
+  $local_tasks['oauth2_token.settings_tab']['base_route'] = 'admin.access_control';
+  $local_tasks['oauth2_token.settings_tab']['weight'] = 1;
+
+  $local_tasks['entity.consumer.collection']['base_route'] = 'admin.access_control';
+  $local_tasks['entity.consumer.collection']['weight'] = 2;
+
+  $local_tasks['entity.user_role.collection']['base_route'] = 'admin.access_control';
+  $local_tasks['entity.user_role.collection']['weight'] = 3;
+
+  $local_tasks['entity.oauth2_token.collection']['base_route'] = 'admin.access_control';
+  $local_tasks['entity.oauth2_token.collection']['weight'] = 4;
+
+  $local_tasks['entity.user.collection']['title'] = t('Users');
+  $local_tasks['entity.user.collection']['base_route'] = 'admin.access_control';
+  $local_tasks['entity.user.collection']['weight'] = 5;
+
+  $local_tasks['entity.node_type.collection']['title'] = t('Content Types');
+  $local_tasks['entity.node_type.collection']['base_route'] = 'admin.content_models';
+  $local_tasks['entity.node_type.collection']['weight'] = 1;
+
+  $local_tasks['entity.media_type.collection']['title'] = t('Media Types');
+  $local_tasks['entity.media_type.collection']['base_route'] = 'admin.content_models';
+  $local_tasks['entity.media_type.collection']['weight'] = 3;
+
+  // If the custom blocks module is enabled, complete some addtional local task
+  // updates so that we see the block types under content model and block
+  // content under content.
+  if ($moduleHandler->moduleExists('block_content')) {
+    $local_tasks['entity.block_content_type.collection']['title'] = t('Block Types');
+    $local_tasks['entity.block_content_type.collection']['base_route'] = 'admin.content_models';
+    $local_tasks['entity.block_content_type.collection']['parent_id'] = NULL;
+    $local_tasks['entity.block_content_type.collection']['weight'] = 2;
+
+    $local_tasks['entity.block_content.collection']['title'] = t('Blocks');
+    $local_tasks['entity.block_content.collection']['base_route'] = 'system.admin_content';
+    $local_tasks['entity.block_content.collection']['weight'] = 0;
+  }
+}
+
+/**
+ * Implements hook_menu_links_discovered_alter().
+ */
+function acquia_cms_headless_ui_menu_links_discovered_alter(array &$links) {
+  $moduleHandler = \Drupal::service('module_handler');
+  $links['system.themes_page']['enabled'] = FALSE;
+  $links['system.admin_structure']['enabled'] = FALSE;
+
+  foreach ($links as &$link) {
+    if (isset($link['icon'])) {
+      $icon = \Drupal::service('extension.path.resolver')->getPath('module', $link['provider']) . '/' . $link['icon'];
+      $icon = \Drupal::service('file_url_generator')->generateAbsoluteString($icon);
```

Comment:
@modularlab Can we use other alternative for service: `file_url_generator`. Because we support Drupal core 9.2 & 9.3. The service `file_url_generator` was introduced in 9.3, so this would give error with Drupal core 9.2.

Reply:
FYI, by using file_create_url() and file_url_transform_relative() over file_url_generator, the [Strict deprecated code scan](https://app.travis-ci.com/github/acquia/acquia_cms/jobs/567189198)  fails.  I'm going to revert this back.